ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/route.tsx

```diff
@@ -137,6 +138,7 @@ const Layout = () => {
           title: 'Job Configurations',
           to: '/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/config',
           params,
+          icon: () => <CalendarCheck className='h-4 w-4' />,
```

Comment:
Commit message nit: At this point, "(run) configuration" should still say "job configuration".